qcog_python_client/qcog/qcogasync.py

```diff
@@ -136,7 +136,10 @@ async def preloaded_training_parameters(self, guid: str) -> AsyncQcogClient:
         return self
 
     async def preloaded_model(self, guid: str) -> AsyncQcogClient:
-        """Retrieve a preexisting model.
+        """Retrieve a preexisting trained model.
+
+        If you are working on a Pytorch model, you need to preload the
```

Comment:
So I guess I'm thinking of the user flow. It seems a bit confusing to force an extra call for the pytorch model path. Why not have an optional parameter that they can put the pytorch model name and do the call within the model fetch?